### PR TITLE
Feature-components

### PR DESCRIPTION
**Pull request by maximoortelli**
### **_Feature-Components_**:
_In this project, i should continue with the development of the Math Magicians app. I will develop a React component that will hold the core functionality: a calculator._

- [x] I have removed all the boilerplate from Create React App (text, images, styles).
- [x] I have created a new directory called components.
- [x] Inside components, i have created a new Calculator.js file.
- [x] In Calculator.js, i have created a React component that matches this design:
calculator

- [x] I have Implemented the design only (JSX and CSS).
   > I have developed just the UI of the calculator; the math logic to make it work will be added in the next project.
- [x] I have used functional components.
- [x] I have created at least 1 child component.
- [x] Once the Calculator is ready, import it and use it in your top-level component: App.js.
- [x] Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js
Comment (joskalenda) [inline src/components/Calculator.js:1]: - [x] Nice dear @maximoortelli, you have done a good job but I think you can still do better, see the requirement states that your design should match the snapshot, however, I found some points of improvement which I will point out below:
- [x] Your design is on the left and the template to follow is on the right
- comparing the 2, I think you are starting your buttons with the wrong number, you should start with 7 as in the image on the right, and so on for the other column
- You have used `/` for division which is right too but not in the requirement, you should use the correct symbole
- Your other 2 signs are not in the right place compared to the original design
- and to finish, you are not using the same color, I think your design is using yellow while in the design template, it is orange
- Please fix these and resubmit your work

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @maximoortelli , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Good design
- [ ].. ✅ Green linters
- [ ].... ✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js
Comment (maximoortelli) [reply src/components/Calculator.js:1]: Done, thanks for the review 👍😃

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (V-Blaze) [inline src/components/Calculator.js:1]: - [OPTIONAL] This was not pointed out by the previous reviewer. As required by Microverse, you are not meant to use any linter disable rule in your codebase. The linters are there to make sure your code follows the best practices and to help you write better codes, you are not meant to disable the checks. I'll suggest you remove this and try to fix the issue. :small_airplane: :smile: :+1:

**Review by V-Blaze — APPROVED** [review]
Hi @maximoortelli  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- UI looks like the provided design 💯
-  Great communication with the reviewer👍
-  Readme looks professional 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**